src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -154,6 +138,11 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
                 this.selectedClinicalAttributeIds.set(SpecialAttribute.StudyOfOrigin, true);
             }
         });
+        onMobxPromise([props.store.samples, props.store.patients], (samples:any[], patients:any[])=>{
```

Comment:
do these get disposed?    i had the idea that component methods could be decorated as reactions.   That would make it possible to dispose them on destruction of component.  it looks like there is some support for this in mobx, but prob not in our version 
https://github.com/mobxjs/mobx-react/issues/181

Reply:
good point - if they dont resolve then they're not disposed

Reply:
Ill log an issue and lets come back to it